### PR TITLE
Add LaTeX support

### PR DESCRIPTION
**Pull request by du-da**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,6 +84,7 @@
 		"tree-sitter-java": "^0.19.1",
 		"tree-sitter-javascript": "^0.19.0",
 		"tree-sitter-json": "^0.19.0",
+		"tree-sitter-latex": "github:latex-lsp/tree-sitter-latex#1666e9780de42a31b1376d32fbe8332e8dd5850f",
```
Comment (pokey) [inline package.json:87]: @wenkokke as newly crowned dependency-management czar for this repo, does this line meet your approval? 😊

**Review by pokey — COMMENTED** [review]
Looks good!  Left one minor question for @wenkokke, but then I'll just pull this one locally to test it out and then merge / deploy

File: package.json (hunk 1 above)
Comment (wenkokke) [reply package.json:87]: Looks great! Very happy for latex support!

**Comment by wenkokke** [comment]
@du-da Do you have any LaTeX input commands which you use? I am still working on my own LaTeX input commands, so I would be happy to see yours!

**Comment by du-da** [comment]
I currently use a stripped-down version of [this repo](https://github.com/mrob95/mathfly-talon/tree/master).

When I was just searching for the repository from which I stole the respective files (via [search.talonvoice.com](https://search.talonvoice.com/search/?q=latex&fold_case=auto&regex=false&context=true)), I stumbled across the following repo, which at first glance might also be interesting: https://github.com/RichardHladik/knausj_talon/tree/master/mine

**Comment by pokey** [comment]
@wenkokke @du-da there's also [this fork of mathfly](https://github.com/ma-anwar/mathfly) which might be worth checking out as well

**Review by pokey — APPROVED** [review]

**Pull request merged**